jdaviz/cli.py

```diff
@@ -28,6 +28,8 @@ def main(filename, layout='default', browser='default', theme='light',
         The path to the file to be loaded into the Jdaviz application.
     layout : str, optional
         Optional specification for which configuration to use on startup.
+    instrument: str, optional
```

Comment:
This also needs updating to state only valid choices.
```suggestion
    instrument : str, optional
```